src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -233,6 +225,7 @@ void fieldRelativeDrive(double xPower, double yPower, MirrorableRotation2d targe
      */
     void fieldRelativeDrive(double xPower, double yPower, double thetaPower) {
         final ChassisSpeeds speeds = selfRelativeSpeedsFromFieldRelativePowers(xPower, yPower, thetaPower);
+        currentFieldRelativeTargetAngle = new MirrorableRotation2d(RobotContainer.POSE_ESTIMATOR.getCurrentEstimatedPose().getRotation(), false);
```

Comment:
do this when you initialize drive, not each time you do a normal field relative drive